scala-setup.sh

```diff
@@ -11,15 +11,19 @@ if [ $# -ge 1 ] && [ "$1" = "--version" ] ; then
   fi
 fi
 
-SCALA_CLI_BASE_URL="https://github.com/VirtusLab/scala-cli/releases/download/v$SCALA_CLI_VERSION/"
+SCALA_CLI_BASE_URL="https://github.com/VirtusLab/scala-cli/releases/download/v$SCALA_CLI_VERSION"
```

Comment:
slash here is not needed, otherwise `URL` variable below contains two `//` (you can easily check by echoing the URL)